schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/errors/UnsupportedSchemaTypeException.java

```diff
@@ -18,7 +18,9 @@
 /**
  * Indicates that schema type is invalid.
  */
-public class UnsupportedSchemaTypeException extends Exception {
+public class UnsupportedSchemaTypeException extends RuntimeException {
```

Comment:
Do we really need to be making RuntimeExceptions, its a lot harder for end users to handle as they're easily left off interfaces etc. We should be avoiding IMO runtime exceptions